DuggaSys/accessed.js

```diff
@@ -130,9 +132,72 @@ function hideEditUserPopup(id) {
 	$("#editUser").css("display", "none");
 }
 
+function hideAddUserPopup() {
+	$("#addUser").css("display", "none");
+}
+
+function hideRemoveUserPopup() {
+	$("#removeUser").css("display", "none");
+}
+
 //----------------------------------------------------------------------------
 //-------------==========########## Commands ##########==========-------------
 //----------------------------------------------------------------------------
+function addUserToCourse() {
+	let input = document.getElementById('addUsernameAdd').value;
+	let term = $("#addTermAdd").val();
+	$.ajax({
+		type: 'POST',
+		url: 'accessedservice.php',
+		data: {
+			opt: 'RETRIEVE',
+			action: 'USER',
+			username: input
+		},
+		success: function(response) {
+			userJson = response.substring(0, response.indexOf('{"entries":'));
+			let responseData = JSON.parse(userJson);
+			let uid = responseData.user[0].uid;
+			AJAXService("USERTOTABLE", {
+				courseid: querystring['courseid'],
+				uid: uid,
+				term: term,
+				coursevers: querystring['coursevers'],
+				action: 'COURSE'
+			}, "ACCESS");
+		},
+		error: function(xhr, status, error) {
+			console.error("Error", error);
+		}
+	});
+	hideAddUserPopup();
+}
+function removeUserFromCourse() {
+	let input = document.getElementById('addUsernameRemove').value;
```

Comment:
Same here, could / should be changed to removeUsername, not addUsernameRemove, since were not adding anything this id seems a little odd.